subscription_platform/explores/daily_active_logical_subscriptions.explore.lkml

```diff
@@ -25,6 +25,14 @@ explore: daily_active_logical_subscriptions {
     ]
   }
 
+  join: subscription_services {
+    from: daily_active_logical_subscriptions__subscription__services
+    sql_table_name: UNNEST(daily_active_logical_subscriptions.subscription.services) ;;
+    sql_on: TRUE ;;
```

Comment:
The `sql` parameter is [deprecated](https://cloud.google.com/looker/docs/reference/param-explore-join-sql), and with this alternative approach Looker requires either `sql_on` or `foreign_key` to be defined, so we have to give it some join condition even though we don't want to filter the join at all.